modules/meilisearch/meilisearch_test.go

```diff
@@ -0,0 +1,87 @@
+package meilisearch_test
+
+import (
+	"context"
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/meilisearch"
+	"io"
+	"net/http"
+	"net/url"
+	"testing"
+)
```

Comment:
Ditto: the imports will be modified after running `make lint` from the root of the module.

Reply:
I have run this command